catroweb.yaml

```diff
@@ -372,6 +368,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ProjectsResponse'
+          headers:
+            x-Project-Hash:
```

Comment:
consistency. Keep the X LARGE ;)

Reply:
...fun fact the header in my test implementation is correct but only the documentation is wrong ;)